pkg/podman/podman.go

```diff
@@ -81,7 +82,10 @@ func (o *PodmanCli) PlayKube(pod *corev1.Pod) error {
 	}()
 	if err = cmd.Wait(); err != nil {
 		if exiterr, ok := err.(*exec.ExitError); ok {
-			err = fmt.Errorf("%s: %s", err, string(exiterr.Stderr))
+			out := strings.Split(podmanOut, "\n")
+			// the last line is an empty new line; so we revert to the second last line for error
+			errLine := out[len(out)-2]
+			err = fmt.Errorf("%s: %s\n%s", err, string(exiterr.Stderr), errLine)
```

Comment:
Why do we want to display only the last non-blank line, and not the entire content?
This seems very specific to a use-case where the error message would be in the last non-blank line.

Reply:
On a second thought, the last line (err line) usually sums up the core issue, so showing just that should be fine. WDYT?

Reply:
I would prefer for the first versions to display the complete output, to have the maximum of information. If we consider it is too much, we could reduce it